examples/errors/mainwindow_funcs.cpp

```diff
@@ -37,15 +37,19 @@ MainWindow::MainWindow() : QMainWindow()
 
     dataEntity_ = ui_.TestingWidget->addData1D("Sin");
     dataEntity_->setData(xValues_, yValues_, uniformErrors_);
+    dataEntity_->setSymbolStyle(Mildred::StyleFactory1D::SymbolStyle::Triangle);
 
     ui_.StyleCombo->addItem(QString("Stick"));
     ui_.StyleCombo->addItem(QString("T-Bar Stick"));
-
+    ui_.StyleCombo->addItem(QString("Triangle"));
     ui_.WidthSpin->setValue(10.0);
+    ui_.TriangleWidth->setValue(12.0);
 
     connect(ui_.WidthSpin, SIGNAL(valueChanged(double)), this, SLOT(setErrorBarSize(double)));
+    connect(ui_.TriangleWidth, SIGNAL(valueChanged(double)), this, SLOT(setSymbolSize(double)));
```

Comment:
The size parameter for our symbols needs to be general, so the `ui_.TriangleWidth` spinbox would be better renamed to:
```suggestion
    connect(ui_.SymbolSizeSpin, SIGNAL(valueChanged(double)), this, SLOT(setSymbolSize(double)));
```